src/openms/include/OpenMS/CONCEPT/Constants.h

```diff
@@ -482,6 +482,12 @@ namespace OpenMS
               String
       */
       inline const std::string   NUM_OF_MASSTRACES = "num_of_masstraces";
+
+      /** User parameter name for the total number of data points (peaks) in a feature. (Required for MQEvidenceExporter)
+              String
+      */
+      inline const std::string   NUMBER_OF_DATAPOINTS = "number_of_datapoints";
```

Comment:
@axelwalter @jpfeuffer we have here number_of and you recently added num_of ... should we try to enforce consistency here?

Reply:
Since this a new meta value we could stay with num_of, which is used already.

Reply:
It is now num_of_datapoints.